<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StringToHexClass</class>
 <widget class="QMainWindow" name="StringToHexClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StringToHex</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>String</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="teString"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Hex</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="teHex"/>
      </item>
      <item>
       <widget class="QComboBox" name="cbEncoding"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="stringtohex.qrc"/>
 </resources>
 <connections/>
</ui>
